core-ui/src/components/Lambdas/constants.js

```diff
@@ -63,6 +63,7 @@ export const LAMBDA_PHASES = {
 export const PRETTY_RUNTIME_NODEJS12_NAME = 'Node.js 12';
 export const PRETTY_RUNTIME_NODEJS14_NAME = 'Node.js 14';
 export const PRETTY_RUNTIME_PYTHON38_NAME = 'Python 3.8';
```

Comment:
```suggestion
export const PRETTY_RUNTIME_PYTHON38_NAME = 'Python 3.8 - Deprecated';
```
See [here](https://github.com/kyma-project/website/pull/744/files#diff-d73d6705bfab87e27c16586adf09ece8d9674bd57bbed011c0d1266d1a509fd5R37)